packages/manager/apps/pci-databases-analytics/src/data/api/database/indexes.api.ts

```diff
@@ -0,0 +1,32 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { ServiceData } from '.';
+import * as database from '@/types/cloud/project/database';
+
+export const getIndexes = async ({
+  projectId,
+  engine,
+  serviceId,
+}: ServiceData) =>
+  apiClient.v6
+    .get(`/cloud/project/${projectId}/database/${engine}/${serviceId}/index`, {
```

Comment:
To prevent from type casting, you can add generic type as `get` type parameter: 
```suggestion
    .get<database.opensearch.Index[]>(`/cloud/project/${projectId}/database/${engine}/${serviceId}/index`, {
```